### DISCUSSION TITLE
Debounce for validation

### DISCUSSION THREAD

**Discussion by MrCasquette** [react-hook-form/react-hook-form#11807] [Ideas]
I look for a solution for debouncing my validation and found nothing on the doc.
I found some use cases that I don't really like on the internet. Should be a good practice to have something like an onChange mode debounced :)

I would like to have everyone's opinion on this topic. Maybe I'm going off track, maybe the solution exist. Let me know 😃

**Comment by jordan-rigo** [comment]
This is something I would like as well.
It would be useful if the validation would be triggered onChange when the user stopped typing.

Is this possible?
